recipes/diff-match-patch/meta.yaml

```diff
@@ -0,0 +1,35 @@
+{% set version = "20121119" %}
+
+package:
+  name: diff-match-patch
+  version: {{ version }}
+
+source:
+  fn: diff-match-patch-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/d/diff-match-patch/diff-match-patch-{{ version }}.tar.gz
+  sha256: 9dba5611fbf27893347349fd51cc1911cb403682a7163373adacc565d11e2e4c
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+
+test:
+  imports:
+    - diff_match_patch
+
+about:
+  home: http://code.google.com/p/google-diff-match-patch/
+  license: Apache 2.0
+  license_file: LICENCE
+  summary: 'The Diff Match and Patch libraries offer robust algorithms to perform the operations required for synchronizing plain text.'
```

Comment:
Well you can use a pipe and then enter a literal block. I am not suggesting changing the text, only that this goes over the line character limit. I guess it isn't a huge deal...